.devcontainer/devcontainer.json

```diff
@@ -1,31 +1,29 @@
-// For more details, see https://aka.ms/devcontainer.json.
 {
   "name": "Mastodon",
   "dockerComposeFile": "docker-compose.yml",
   "service": "app",
   "workspaceFolder": "/workspaces/${localWorkspaceFolderBasename}",
 
-  // Features to add to the dev container. More info: https://containers.dev/features.
   "features": {
     "ghcr.io/devcontainers/features/sshd:1": {}
   },
 
-  // Use 'forwardPorts' to make a list of ports inside the container available locally.
-  // This can be used to network with other containers or the host.
```

Comment:
This looks good overall but can we keep the comments?  they provide some helpful context.

Reply:
We generally don't keep such documentation of external libraries/code/systems in our repo, unless there is something special/unusual going on. I'd much recommend just looking up the official documentation on https://containers.dev.

Reply:
ok, that makes sense to me.  thx.  will approve